packages/x-charts-pro/src/ChartDataProviderPro/ChartDataProviderPro.tsx

```diff
@@ -82,7 +82,7 @@ ChartDataProviderPro.propTypes = {
   children: PropTypes.node,
   /**
    * Color palette used to colorize multiple series.
-   * @default rainbowSurgePalette
+   * @default blueberryTwilightPalette
```

Comment:
Why was this changed from `rainbowSurgePalette` to `blueberryTwilightPalette`? Rainbow surge is the new one and should be the default

Reply:
Not sure, it wasn't intended. 

Before I had the update to the default color in this PR as well, but when I moved it into a different one I must have messed something up. It's fixed now!